src/lib-components/BlockCardWithImage.vue

```diff
@@ -130,10 +140,13 @@ const parsedDateFormat = computed(() => {
         <MoleculePlaceholder
           class="molecule"
           aria-hidden="true"
+          :aspect-ratio="imageAspectRatio"
```

Comment:
does this have any effect? I don't think this SVG has any implementation for aspect-ratio?

Reply:
yeah, I'll take that out. I was having an issue with it expanding vertically so I tried this to see if it worked. I ended up handling it in the CSS. I forgot to take this out.